app/renderer/assets/locales/en-US/rex.json

```diff
@@ -30,11 +30,11 @@
   "rex_interface_menu_lend_rex": "Lend Rex",
   "rex_interface_menu_rent_resources": "Rent Resources",
   "rex_interface_not_voting_or_proxying": "You appear to not be meeting the minimum criteria to use this feature. In order to buy Rex, you must first either vote for 21 block producers or proxy your votes.",
-  "rex_interface_rent_funding_balance": "{{fundedBalance}} {{chainSymbol}} available in fund.",
```

Comment:
Feel free to disregard these if you had already replaced all the strings on your side :D